code/modules/tips_and_tricks/mobs.dm

```diff
@@ -0,0 +1,8 @@
+// This file contains all mobs tips and tricks and shown when you spawn or your mind is transfered
+/tipsAndTricks/mobs
+    var/list/mobs_list       //list of mobs to which tip can be shown
+    textColor = "#957820"
+
+/tipsAndTricks/mobs/breathing
+    mobs_list = list(/mob/living/carbon/human)
+    tipText = "As a human you need oxygen and a pressure to breath. Comfortable pressure for human is around 101psi. Some gasses like phoron are toxic to you."
```

Comment:
>Phoron
:angry:

Reply:
Also kPa and psi mixup, 697 kPa isn't very comfortable.

Reply:
Fix this.